internal/cmd/operator-sdk/scorecard/cmd.go

```diff
@@ -102,12 +104,108 @@ func (c *scorecardCmd) printOutput(output v1alpha3.TestList) error {
 			return fmt.Errorf("marshal json error: %v", err)
 		}
 		fmt.Printf("%s\n", string(bytes))
+	case "xunit":
+		xunitOutput, err := c.convertXunit(output)
+		if err != nil {
+			return fmt.Errorf("xunit conversion error: %v", err)
+		}
+		bytes, err := xml.MarshalIndent(xunitOutput, "", "  ")
+		if err != nil {
+			return fmt.Errorf("marshal xml error: %v", err)
+		}
+		fmt.Printf("%s\n", string(bytes))
 	default:
 		return fmt.Errorf("invalid output format selected")
 	}
 	return nil
 }
 
+type TestCase struct {
```

Comment:
Add a comment for the public structs. In a follow on, consider moving these to an xunit package close by.